robot/impl/local_robot_test.go

```diff
@@ -3133,7 +3133,7 @@ func TestCrashedModuleReconfigure(t *testing.T) {
 		test.That(t, os.Unsetenv(rutils.ResourceConfigurationTimeoutEnvVar),
 			test.ShouldBeNil)
 	}()
-	test.That(t, os.Setenv(rutils.ResourceConfigurationTimeoutEnvVar, "500ms"),
+	test.That(t, os.Setenv(rutils.ResourceConfigurationTimeoutEnvVar, "2s"),
 		test.ShouldBeNil)
```

Comment:
I think we should remove the timeout and just use the default of `1m`, unless we are specifically testing this timeout configuration setting. Otherwise setting a tighter deadline just increases the chance of flaking.

```suggestion
```

Reply:
Okay, I misunderstood - part of this test relies on a timeout, so this would make the test much longer. I would recommend unsetting the timeout after we expect it to happen.

Reply:
> I would recommend unsetting the timeout after we expect it to happen.

Great idea! Done (I think). Also put in some logic to make sure we still unset that env var even if we fail earlier in the test.